src/Faker/Provider/uk_UA/Address.php

```diff
@@ -11,6 +11,10 @@ class Address extends \Faker\Provider\Address
     );
     protected static $buildingNumber = array('##');
     protected static $postcode = array('#####');
+
+    /**
+     * @see https://uk.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%BE%D0%BA_%D0%BA%D1%80%D0%B0%D1%97%D0%BD_%D1%81%D0%B2%D1%96%D1%82%D1%83
```

Comment:
the content of this page and the content of the country list in this class don't match. The idea of having a datasource is to have an external reference where other people can look for changes and update the dataset in the future.